lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -160,11 +180,13 @@ class _FindClosestFacilityFromPointSampleState
         continue;
       }
 
+      // Get the route to the closest facility.
       final closestFacilityIdx = rankedFacilities.first;
       final routeToFacility = results.getRoute(
         facilityIndex: closestFacilityIdx,
         incidentIndex: incidentIdx,
       );
+      // Add the route to the graphics overlay.
       if (routeToFacility != null) {
         final routeGraphic = Graphic(
           geometry: routeToFacility.routeGeometry,
```

Comment:
I can't do the suggestion properly, but you can remove the comma after `symbol; _routeLineSymbol`

Reply:
is it because there are only two parameters?